cake.config

```diff
@@ -8,8 +8,8 @@ LoadDependencies=false
 
 [Paths]
 Tools=./tools
```

Comment:
@paulomorgado wouldn't be change also be required here?

Reply:
Fixed!